site/src/atomics/border.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Border Atomics
+description: Border related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Border Atomics
+
+Various classes can be used to apply/customize borders on elements.
+
+- `.border`: sets a 1px border on an element
+- `.border-none`: resets border to none
+- `.border-{direction}`: sets a 1px border for that user direction
+- `.border-{direction}-none`: resets border for that user direction
+- `.border-{direction}-{screensize}`: sets a 1px border for that user direction on specific screen size
+- `.border-{direction}-{screensize}-none`: resets border for that user direction on specific screen size
+
+Currently, the only `{screensize}` available is `tablet`.
+
+## Example
+
+Here is an example of simple `border` class usage:
+
+```html
+<div class="padding-s border">
+	<p>Text block</p>
+</div>
+```
+
+Here is an example of applying border to specific side of the element:
```

Comment:
I'm wondering if it would make sense to use `user direction` instead of `side` to match the wording above.

Reply:
Hmm I'd better update it above to say element side 🤔

Reply:
updated